Sources/BraintreeShopperInsights/Models/CreateCustomerSessionMutationGraphQLBody.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+
+// swiftlint:disable nesting
+/// The POST body for the GraphQL mutation `CreateCustomerSession`
+struct CreateCustomerSessionMutationGraphQLBody: Encodable {
```

Comment:
Can we add a few unit tests for this?

Reply:
yep added in 9e6acc4ed40ae2ab74a6ff1e180f3f0401834df7